crates/swc/src/config/mod.rs

```diff
@@ -932,6 +935,9 @@ pub struct JscExperimental {
     /// This requires cargo feature `plugin`.
     #[serde(default)]
     pub plugins: Option<Vec<PluginConfig>>,
+    /// If true, keeps import assertions in the output.
+    #[serde(default)]
+    pub import_assertions: bool,
```

Comment:
Can you change this to `keep_import_assertions`?

Reply:
Done, and updated the OP.